Mathlib/Data/Nat/Bitwise.lean

```diff
@@ -296,6 +296,30 @@ theorem testBit_two_pow (n m : ℕ) : testBit (2 ^ n) m = (n = m) := by
     simp [h]
 #align nat.test_bit_two_pow Nat.testBit_two_pow
 
+@[simp]
+lemma pred_bit0 {x : ℕ} (h : x > 0) : pred (bit0 x) = bit true (pred x) := by
```

Comment:
Is `bit true` equal to `bit1`? If so I think that might be the most sensible statement.

Reply:
They are equal, yes. I tried to avoid `bit1` sering as it is deprecated. Then again, it is still the simp normal form (which is what forced me to use `bit0`), so it makes sense to change.

Reply:
We have the applied version as [bit_false](https://leanprover-community.github.io/mathlib4_docs/Mathlib/Data/Nat/Bitwise.html#Nat.bit_false) and bit_true, would those suffice?

Reply:
That works better, I have pushed to use these lemmas.

Reply:
Perfect, then I'm marking this as resolved, since we made the suggested change